iabgpp-encoder/src/main/java/com/iab/gpp/encoder/base64/CompressedBase64UrlEncoder.java

```diff
@@ -0,0 +1,38 @@
+package com.iab.gpp.encoder.base64;
+
+import java.util.Arrays;
+
+public class CompressedBase64UrlEncoder extends AbstractBase64UrlEncoder {
+
+  private static CompressedBase64UrlEncoder instance = new CompressedBase64UrlEncoder();
+  
+  private CompressedBase64UrlEncoder() {
+    
+  }
+  
+  public static CompressedBase64UrlEncoder getInstance() {
+    return instance;
+  }
+  
+  @Override
+  protected String pad(String bitString) {
+    char[] chars1 = null;
+    if(bitString.length() % 8 > 0) {
```

Comment:
Hi, @iabmayank 
We have another question here regarding the GPP encoding and decoding. Why must the length of bitString be a multiple of 8 first? We recently encountered some GPP strings like DBABzw~1YYN~BVQVAAAAAg that don't meet this padding logic but can still be decoded via the [IAB GPP Encoder / Decoder](https://iabgpp.com/#).
Could you please share your thoughts on this?
Thanks in advance.